browser/fxr/content/fxr-fullScreen.js

```diff
@@ -42,6 +42,38 @@ var FullScreen = {
       return;
     }
 
+    // Get the mozVideoProjection query string if there is one, and use it to set the projection mode for FxR
+    // See: https://github.com/MozillaReality/FirefoxReality/wiki/Media-Playback
+    // NOTE: Currently "Stereo 180 Left to Right" and "Stereo 180 Top to Bottom" are unsupported
+    var currentFullscreenURI = document.fullscreenElement.currentURI;
+    if (currentFullscreenURI &&
+        currentFullscreenURI.query
+    )
+    {
+      var mozVideoProjectionIndex = currentFullscreenURI.query.indexOf("mozVideoProjection=");
+      if (mozVideoProjectionIndex != -1)
+      {
+        var projectionMode = currentFullscreenURI.query.substring(currentFullscreenURI.query.indexOf("mozVideoProjection=") + 19);
```

Comment:
a few things: either
make `mozVideoProjection=` a `const` variable, and then use its length property (instead of magic number 19 and repeated string)
or
use URLSearchParams to get the value